g3doc/user_guide/tutorials/docker-in-gvisor.md

```diff
@@ -6,6 +6,17 @@ applications.
 In gVisor, all basic docker commands should function as expected. The host
 network driver and the bridge network driver are tested and supported.
 
+### Limitations
+
+- `dockerd` inside gVisor needs to be executed with flags
+  `--iptables=false --ip6tables=false` and additional network setup is needed, check
+  [images/basic/docker/start-dockerd.sh](https://github.com/google/gvisor/blob/master/images/basic/docker/start-dockerd.sh)
+  for the reference
+- because iptables disabled, `docker run --expose=` does not expose the port;
+  if a nested container needs to expose ports, inside gVisor use `docker run --network=host`
+- docker 28 may not work
+- docker compose does not work ([#7469](https://github.com/google/gvisor/issues/7469))
```

Comment:
the only issue with docker compose is when it uses bridge network driver when docker compose run, there is DNS issue. please see https://github.com/google/gvisor/blob/29259326ccc9423d94cc0d71309cf557073eb7c6/test/image/image_test.go#L676

docker compose mostly works in gVisor

Reply:
I removed the phrase about docker compose, because I don't understand how significant the issue with DNS.

If it is significant, I can add it back.